app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.kt

```diff
@@ -104,6 +104,7 @@ class ChatViewHolder(view: View, clickListener: MessageViewHolder.ClickListener,
                                 thumbsUp?.setImageResource(R.drawable.thumbs_up_solid)
                                 model.skillLocation?.let { location -> rateSusiSkill(Constant.POSITIVE, location, context) }
                                 setRating(true, true)
+                                Toast.makeText(context,"Thank you for responding", Toast.LENGTH_SHORT).show()
```

Comment:
Is this change for this PR?

Reply:
Yes it is! Because I thought the user might know if the language has changed if there's no toast message ,like an intimation. So I thought it is important

Reply:
I can see same changes in #2121 . Why so?

Reply:
That is probably because I couldn't cherry pick and squash commits. I shall see that other pull request commits don't enter here.
May I ask you, Can closing the other pull request solve the issue for now? Because this one has the commit of other pull request as well.